lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -236,23 +236,41 @@ function paraneue_dosomething_preprocess_reportback_gallery(&$vars, $count = 6)
     'random' => TRUE,
   ];
 
+  // If the Rogue feature flag is on, grab reportbacks from Rogue.
+  if (variable_get('rogue_collection', FALSE)) {
+    // Hit the GET /reportbacks endpoint in Rogue with campaign_id filter.
+    // On the Rogue side, it will only return 'approved', which will be shown in the gallery.
+    $promotedReportbackItems = dosomething_rogue_get_reportback_items(['campaign_id' => $gallery['campaign_id']])['data'];
 
-  // # Promoted Reportbacks:
-  try {
-    $promotedReportbackItems = (new ReportbackItem)->find($filters);
-  }
-  catch (Exception $error) {
-    $promotedReportbackItems = [];
+    // Since Rogue only sends back approved reportbacks, we can just set inital_promoted_count, total_promoted and total_approved to count($promotedReportbackItems)to not make same API calls over and over.
+    // inital_promoted_count, total_promoted, and total_approved all the same now in Rogue world.
+    $gallery['initial_promoted_count'] = count($promotedReportbackItems);
+
+    $gallery['total_promoted'] = count($promotedReportbackItems);
+
+    // // # Approved Reportbacks:
+    $gallery['total_approved'] = count($promotedReportbackItems);
   }
-  $gallery['initial_promoted_count'] = count($promotedReportbackItems);
-  $gallery['total_promoted'] = dosomething_reportback_get_reportback_items_total_by_filters(['nid' => $gallery['campaign_id'], 'status' => 'promoted']);
+  else {
+    // # Promoted Reportbacks:
+    try {
+      $promotedReportbackItems = (new ReportbackItem)->find($filters);
+    }
+    catch (Exception $error) {
+      $promotedReportbackItems = [];
+    }
+    $gallery['initial_promoted_count'] = count($promotedReportbackItems);
+
+    $gallery['total_promoted'] = dosomething_reportback_get_reportback_items_total_by_filters(['nid' => $gallery['campaign_id'], 'status' => 'promoted']);
 
-  // # Approved Reportbacks:
-  $gallery['total_approved'] = dosomething_reportback_get_reportback_items_total_by_filters(['nid' => $gallery['campaign_id'], 'status' => 'approved']);
+    // # Approved Reportbacks:
+    $gallery['total_approved'] = dosomething_reportback_get_reportback_items_total_by_filters(['nid' => $gallery['campaign_id'], 'status' => 'approved']);
+  }
 
   // # Build the full initial gallery:
 
   // Collect Reportbacks to fulfill desired amount.
+  // @TODO: delete this comment but the below should be fine if we're okay with grabbing the placeholder still from Phoenix?
```

Comment:
Per comment, the below should be fine if we're okay with grabbing placeholders still from Phoenix, correct?